devdocs/process.md

```diff
@@ -271,6 +276,13 @@ The meaning of label group prefixes:
   Not to be applied for issues with a priority `p.high` or above.
 * `d.Committers`: More difficult issues that are better left for committers or more senior developers.
 
+**Effort**
+
+This label is used to indicate how much effort is expected for (or was spent on)
+an issue.
+
+`e.1` is roughly equal to an hour of work, `e.2` is two hours of work, and so on.
```

Comment:
If the effort value is going to be a running number, how many such labels would we have? 
To address this, shall we say that if the expected effort exceeds the max effort label, we should break it down into smaller issues